.gitlab-ci-main.yml

```diff
@@ -19,14 +19,18 @@ variables:
 .static_template: &static_template
   before_script:
     - |
-      docker network prune -f && docker network create amazeeio-network
-      docker login -u gitlab-ci-token -p $CI_JOB_TOKEN $CI_REGISTRY
+      if [ "${GOVCMS_FAST_LINTING-FALSE}" = "TRUE" ] ; then
+        composer update
```

Comment:
Any reason this is an `update` and not `install` as per the dockerised approach?

Reply:
Hmm good pick up. If the composer.lock is out of date the prelaunch would fail so there is no reason why it should fail.